examples/tracing/kernel_cache_alloc_free.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python
+#
+# kernel_cache_alloc_free.py	Kernel memory (alloc-free) monitoring tool.
+#		
+#
+# Written as a basic tool example of using ePBF to track of kernel cache allocation and deallocation process.
+#
+#
+# Copyright (c) 2019 Jugurtha BELKALEM.
+# Licensed under the Apache License, Version 2.0 (the "License")
+#
+# 14-Jan-2019 Jugurtha BELKALEM Created this.
+
+from bcc import BPF
+import ctypes as ct
+# prog will store the eBPF C program
+counter = 0;
+prog = """
+struct memory_allocations {
+     u64 timestamp;
+     u64 call_site;
+     u64 ptr;
+     u64 bytes_req;
+     u64 bytes_alloc;
+     u32 gfp_flags;
+     char command[30];
+};
+
+/* Remember : When using tracepoints we need to make use of /sys/kernel/debug/tracing/events directory.
+
+For example : the parameters of the function kmem_cache_alloc are defined in /sys/kernel/debug/tracing/events/kmem/kmem_cache_alloc/format
```

Comment:
No need for the empty line in the above. Actually the above comments are not needed. But if you really want, you can just say "the tracepoint kmem/kmem_cache_{alloc,free} format is defined at /sys/kernel/debug/tracing/events/kmem/kmem_cache_{alloc,free}/format and they are automatically retrieved by bcc."